tests/integration/session/test_multiple_login.py

```diff
@@ -13,6 +13,7 @@ def setUp(self):
 
         self.cache = {}
 
+    # pylint: disable=arguments-renamed
```

Comment:
I suggest fixing the issue.

Reply:
Does this need to be changed? T because `schema_name="test_textfield"` is a default value, which also means that `client` has to be before it, which pylint is thinking that `schema_name` has been renamed to` client`

Reply:
Removed the `schema_name` default value and swapped it with the `client`. This has removed the pylint error.

Reply:
Oops forgot to update the tests.

Reply:
Can `# pylint: disable=arguments-renamed` be removed now?

Reply:
Removed 👍